### PR TITLE
[MacOS] Add GraalVM to macOS 1015 and 11 images

### PR DESCRIPTION
**Pull request by shilovmaksim**
# Description
Installing GraalVM to macOS 1015 and 11 images via homebrew referencing https://github.com/graalvm/homebrew-tap

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by miketimofeev** [comment]
@shilovmaksim why do we skip installation for macOS-12? Does graalvm not work there?

File: images/macos/provision/core/graalvm.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+echo Installing GraalVM...
+brew install --cask graalvm/tap/graalvm-ce-java11
+
+exportPath=$(echo /Library/Java/JavaVirtualMachines/graalvm-ce-java*/Contents/Home/bin)
+echo "Setting up GRAALVM_11_ROOT variable to ${exportPath}"
+echo "export GRAALVM_11_ROOT=${exportPath}" >> "${HOME}/.bashrc"
+
+# GraalVM not being signed and notarized yet on macOS Catalina
+if ! is_Catalina; then
```
Comment (miketimofeev) [inline images/macos/provision/core/graalvm.sh:12]: Why is not if the comment says about Catalina?

**Comment by shilovmaksim** [comment]
> @shilovmaksim why do we skip installation for macOS-12? Does graalvm not work there?

I just followed the request. MacOS 12 was not specified.

File: images/macos/provision/core/graalvm.sh (hunk 1 above)
Comment (shilovmaksim) [reply images/macos/provision/core/graalvm.sh:12]: Indeed. Typo.

**Comment by miketimofeev** [comment]
> > @shilovmaksim why do we skip installation for macOS-12? Does graalvm not work there?
> 
> I just followed the request. MacOS 12 was not specified.

this is because macOS-12 is not available for public yet and we don't have this checkbox in the issue template either

**Comment by shilovmaksim** [comment]
> > > @shilovmaksim why do we skip installation for macOS-12? Does graalvm not work there?
> > 
> > 
> > I just followed the request. MacOS 12 was not specified.
> 
> this is because macOS-12 is not available for public yet and we don't have this checkbox in the issue template either

Correct. Should I?

**Review by al-cheb — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**